packages/core/test/controls/numericInputTests.tsx

```diff
@@ -57,6 +57,12 @@ describe("<NumericInput>", () => {
             const value = component.state().value;
             expect(value).to.equal("1");
         });
+
+        it("sets allowNumericCharactersOnly to true by default", () => {
+            const component = mount(<NumericInput />);
+            const value = component.props().allowNumericCharactersOnly;
+            expect(value).to.be.true;
```

Comment:
i agree with @leebyp here--this is not an important feature to test. it's a trivial test that can't catch bugs in our code because there's no logic being exercised here. it also doesn't increase code coverage.